department.rb

```diff
@@ -0,0 +1,37 @@
+# require './employee'
+
+class Department
+attr_reader :name, :roster
+  def initialize(name)
+    @name = name
+    @roster = []
+    @employees = []
```

Comment:
You have a @roster variable AND an @employees variable, but it looks like that's causing you problems.  You put employees IN to the @roster variable on line 12, but you expect them to be in @employees on line 26.  Did department_raise do anything?